terraform/main.tf

```diff
@@ -0,0 +1,117 @@
+provider "aws" {
+  region = "us-east-1"
+  profile = "serverless"
+}
+
+data "aws_vpc" "default" {
+  vpc_id = "vpc-0d60ef77"
```

Comment:
This is throwing an error for me.

`An argument named "vpc_id" is not expected here`

Reply:
Weird. It's working fine here.